interfaces/Portal/src/app/components/physical-card-popup/physical-card-popup.component.html

```diff
@@ -64,46 +64,6 @@
       </ion-label>
```

Comment:
@diderikvw 

Hmm since you pushed to the branch I started,  I am not able to approve these changed, since I now co-authored it. It also makes it a bit harder to see what you changed. I guess we can just ask a third person to approve it later

Reply:
I do not understand what you mean.

Reply:
>Hmm since you pushed to the branch I started, I am not able to approve these changed, since I now co-authored it. 

In this case, the person who opened the PR (let's call them User A) started a branch and made some changes. Another person (User B) then pushed additional changes to the same branch.

Because User B made changes to the branch, GitHub considers them a co-author of the PR. This means User B can't approve the PR, because GitHub doesn't allow authors or co-authors of a PR to approve their own changes. This is to ensure that at least one other person has reviewed the changes.

> It also makes it a bit harder to see what you changed

I have to click and open individual commits instead of looking at files changed of a PR

Reply:
I think you also squashed all commits. This makes it a harder for me to see what changed chronologically: 1) what changes were made since my commits 2) What changes were made since my last review.

Preferably I would only review option 2 this time. But squashing the commits to one made this hard

Reply:
Squash before rebase is convenient, but in this case I should have left the commit after your (1st) review in tact, so you can see the changes I made afterwards.